docs/document-processing-logging-guide.md

```
@@ -11,61 +11,60 @@ Comprehensive logging has been added to the `DocumentCreatedHandler` and `Docume
-| ?? | Scheduling | Job being scheduled with delay |
-| ?? | Exit | Handler exiting |
-| ?? | Exception | Unhandled exception |
-| ?? | Start | Processor/job starting |
-| ?? | Fetch | Fetching data |
-| ?? | Lookup | Finding/searching |
-| ?? | Execute | Running operation |
+## Log Message Prefix Conventions
+
+**Handler (`DocumentCreatedHandler`):** Uses plain text prefixes without emojis:
+- `HANDLER_ENTRY`, `HANDLER_ENQUEUE_IMMEDIATE`, `HANDLER_ENQUEUED`, `HANDLER_SCHEDULE_DELAYED`, `HANDLER_SCHEDULE_IMMEDIATE`, `HANDLER_SCHEDULED`, `HANDLER_EXIT`, `HANDLER_MAX_RETRIES`, `HANDLER_EXCEPTION`
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)